sklearn/metrics/cluster/_expected_mutual_info_fast.pyx

```diff
@@ -54,12 +54,12 @@ def expected_mutual_information(contingency, int n_samples):
     start = np.maximum(start, 1)
     end = np.minimum(np.resize(a, (C, R)).T, np.resize(b, (R, C))) + 1
     # emi itself is a summation over the various values.
-    emi = 0
+    emi = 0.0
```

Comment:
emi is defined as `DOUBLE`, not sure why you've added the `.0`

Reply:
it is more explicit if you did not read the definition :)